scripts/analysis/amwg_table.py

```diff
@@ -295,17 +302,39 @@ def _spatial_average(indata):
 
 #####
 
-def _write_html(f, out):
+def _write_html(f, out,case_name,case_name_list):
+    if case_name == case_name_list[0]:
+        case = "Test"
+    else:
+        case = "Control"
```

Comment:
In multi-case ADF runs the last entry in the case names list will be the baseline, and everything else will be a test case. I am not sure how you want to handle it at the moment (given that this function doesn't appear ready for multi-case usage yet?), but I would recommend setting the "Control" case to only be when `case_name = case_name_list[-1]`.